cmd/go-clang-gen/main.go

```diff
@@ -203,6 +203,32 @@ func prepareStructMembers(s *gen.Struct) {
 			}
 		}
 	}
+
+	// If the struct has two member variables, one is an array and the other a plain int/uint with size/length/count/len as its name we connect them two handle a slice.
+	if len(s.Members) == 2 {
+		if ((s.Members[0].Type.PointerLevel == 1 && s.Members[1].Type.PointerLevel == 0) || (s.Members[0].Type.PointerLevel == 0 && s.Members[1].Type.PointerLevel == 1)) &&
+			!s.Members[0].Type.IsArray && !s.Members[1].Type.IsArray &&
+			!s.Members[0].Type.IsSlice && !s.Members[1].Type.IsSlice &&
+			((gen.IsInteger(&s.Members[0].Type) && !gen.IsInteger(&s.Members[1].Type)) || (!gen.IsInteger(&s.Members[0].Type) && gen.IsInteger(&s.Members[1].Type))) {
+			var a *gen.StructMember
+			var c *gen.StructMember
+
+			if s.Members[0].Type.PointerLevel == 1 {
```

Comment:
I assume this is for the case len, array or array,len
Consider introducing a method to improve readability